src/Turbo/DependencyInjection/Configuration.php

```diff
@@ -47,17 +47,6 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->end()
                 ->end()
                 ->scalarNode('default_transport')->defaultValue('default')->end()
-                ->arrayNode('mercure')
```

Comment:
This part hasn't been copied in the bundle. Is it on purpose?

Reply:
yep, there is nothing to configure, see symfony/mercure-bundle#44